app/views/docs/production.phtml

```diff
@@ -1,10 +1,10 @@
-<p>Appwrite default setup is designed to help you get started quickly with using the Appwrite server. To run Appwrite successfully in a production environment, you should follow a few basic concepts and best practices. This document assumes you have some very basic understanding of Docker and Docker Compose command-line tools.</p>
+<p>Appwrite's default setup is designed to help you start building with Appwrite quickly. To succeed with Appwrite in a production environment, you should follow a few basic concepts and best practices. This document assumes you have some basic understanding of Docker and Docker Compose command-line tools.</p>
 
 <h2><a href="/docs/production#encryption" id="encryption">Encryption</a></h2>
 
-<p>By default, the Appwrite setup doesn’t come with a uniquely generated encryption key. This key is used to store your files and sensitive data like webhook passwords or API keys in a safe way. To take advantage of this feature, you must generate a unique key and set it as the value of the <b>_APP_OPENSSL_KEY_V1</b> environment variable.</p>
+<p>Appwrite does not generate a unique encryption key during setup by default. This key encrypts your files and sensitive data like webhook passwords or API keys to keep them secure. To take advantage of this feature, you must generate a unique key and set it as the value of the <code>_APP_OPENSSL_KEY_V1</code> environment variable.</p>
 
-<p>Make sure to keep this key in a safe place and never make it publicly accessible. There are many online resources with methods of keeping your secret keys safe in your servers.</p>
+<p>Make sure to keep this key in a safe place and never make it publicly accessible. You can reference many online resources about securely storing secrets on your servers.</p>
```

Comment:
Should we explicitly give them some best practices or at least link to a resource?

Reply:
If we say there are resources, we should provide some...Otherwise, I don't think we should say anything at all.

Reply:
I think we should not. I heard ideas around Appwrite Vault, key rotations... That might be the solution? Not sure when this is planned tho. You can ask @eldadfux  or @lohanidamodar for more info.

Reply:
I would say we can just not mention this. If it's out of scope and we don't want to link, we shouldn't leave a statement like this in the docs.

It's annoying to read (you're booting me off on my own?) and doesn't have a purpose.

Reply:
+1 to not mention anything at the moment. This is very coupled to the way your team manages infrastructure and your tools of choice. We could probably write a book about it.